Participants/ari.md

```diff
@@ -0,0 +1,7 @@
+### Hi, I'm [your-github-username]
```

Comment:
Hey @arilivigni , could you replace `[your-github-username]` with your name please?

Reply:
I changed it from Ari to arilivigni which is my github username